src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/ManageController.cs

```diff
@@ -272,6 +274,34 @@ public async Task<IActionResult> SetPassword(SetPasswordViewModel model)
             return this.RedirectToAction(nameof(this.SetPassword));
         }
 
+        [HttpGet]
+        public async Task<IActionResult> DisplayStatistics()
+        {
+            var user = await this._manageService.GetUserAsync(this.User).ConfigureAwait(false);
+            if (user == null)
+            {
+                throw new InvalidOperationException(string.Format(CultureInfo.InvariantCulture, ExceptionsMessages.UserWithIdNotFound, this._accountService.GetUserId(this.User)));
+            }
+
+            // TODO: Test for null (the right way)
+            // TODO: check all questions
+            // DateTime answerDate = this._testStatisticsService.GetCorrectAnswerDateTime(user.Email, 1);
+            string messsageToUser = this._testStatisticsService.GetCorrectAnswerDateTime(user.Email, 1).ToString(CultureInfo.InvariantCulture);
+            if (messsageToUser == null)
+            {
+                messsageToUser = "NoAnswer";
```

Comment:
remember to use resources
I suggest more descriptive text for example "Nie udzieliłeś jeszcze poprawnej odpowiedzi na żadne pytanie"